cmd/gb-vendor/main.go

```diff
@@ -31,23 +31,26 @@ var commands = []*cmd.Command{
 }
 
 func main() {
-	root, err := cmd.FindProjectroot(projectroot)
-	if err != nil {
-		gb.Fatalf("could not locate project root: %v", err)
-	}
-	project := gb.NewProject(root)
-	gb.Debugf("project root %q", project.Projectdir())
-
 	args := os.Args[1:]
-	if len(args) < 1 || args[0] == "-h" {
+
+	switch {
+	case len(args) < 1, args[0] == "-h", args[0] == "-help":
 		fs.Usage()
 		os.Exit(1)
-	}
-
-	if args[0] == "help" {
+	case projectroot == "":
+		gb.Fatalf("don't run this binary directly, it is meant to be run as 'gb vendor ...'")
+	case args[0] == "help":
```

Comment:
I think the idea is that if you run gb-vendor directly it should tell you to run it as 'gb vendor ...' It seems like that should hold for 'gb-vendor help', too. The case in question fires when you run it properly, as 'gb vendor help'.

That said, it's always nicer to be greeted by help text instead of a fatal error log. I've swapped the cases as suggested - it works.